FailedBonusActivationActivity/index.ts

```diff
@@ -1,11 +1,52 @@
-import { Context } from "@azure/functions";
+import * as documentDbUtils from "io-functions-commons/dist/src/utils/documentdb";
+import { getRequiredStringEnv } from "io-functions-commons/dist/src/utils/env";
+import {
+  BONUS_ACTIVATION_COLLECTION_NAME,
+  BonusActivationModel
+} from "../models/bonus_activation";
+import {
+  BONUS_LEASE_COLLECTION_NAME,
+  BonusLeaseModel
+} from "../models/bonus_lease";
+import {
+  ELIGIBILITY_CHECK_COLLECTION_NAME,
+  EligibilityCheckModel
+} from "../models/eligibility_check";
+import { documentClient } from "../utils/cosmosdb";
+import { FailedBonusActivationHandler } from "./handler";
 
-const FailerBonusActivation = async (
-  context: Context,
-  input: unknown
-): Promise<unknown> => {
-  context.log.info(`FailerBonusActivationActivity|INFO|Input: ${input}`);
-  return true;
-};
+const cosmosDbName = getRequiredStringEnv("COSMOSDB_BONUS_DATABASE_NAME");
 
-export default FailerBonusActivation;
+const documentDbDatabaseUrl = documentDbUtils.getDatabaseUri(cosmosDbName);
+
+const eligibilityCheckModel = new EligibilityCheckModel(
```

Comment:
I'm think about making these models singletons (into cosmosdb.ts) what about it ? cc @BurnedMarshal

Reply:
Kinda lower transparency as env variables would hide in the common `cosmosdb.ts` module. I'd rather have a `index.ts`-like file where all objects are built at startup

Reply:
Not a big impact though, so we can easily refactor if we fancy

Reply:
there already are envs in cosmosdb.ts https://github.com/pagopa/io-functions-bonus/blob/master/utils/cosmos.ts#L7 that's right with singletons

Reply:
We should open a chore

Reply:
If we change the pattern of how we use the models inside the functions, we should align these changes even on other projects like `io-functions-app`